### ISSUE TITLE
Make the 'related organisations' page dynamic

### ISSUE THREAD

**Issue by thaliawww-service** [thaliawww/concrexit#808]
In GitLab by rwechel on Jan 22, 2019, 20:37

### One-sentence description

Make the 'related organisations' page dynamic

### Motivation

The board wants regular changes to the related organisations pages. So instead of trying to keep this up-to-date in updates we can make this dynamic.

### Desired functionality

We want one overview page with all organisation Thalia is related too.
That overview page can link to individual pages.

### Suggested implementation

New app 'contacts'. With a model 'Contact' that contains the name, email address and description. Maybe also logos for awesome designs. Other possibilities include adding connections between contacts, but that is not a real requirement and can be further investigated.

**Comment by thaliawww-service** [comment]
In GitLab by @JobDoesburg on Jan 23, 2019, 00:22

Isn't it NINJA (at)rwechel ?

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Jan 23, 2019, 11:22

A few points to think about:
- We do mention these organizations already in the reference document for committees and contacts. Whether that is enough is a different question.
- Who is this page for? I do not think anybody is actively looking for this information on our website.
- We have a [page dedicated to student participation](https://thalia.nu/education/student-participation/). If we would make a page with contacts, it could be merged with the student participation page.
- I would not add WISO to the list, because WISO is only relevant to the board members.
- These organizations tend to change/disappear a lot in a period of a few years, which means that this page needs regular upkeep (especially to keep it in sync and up to date with the reference document).


P.S. @JobDoesburg on their website "Ninja" is used.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Jan 24, 2019, 20:53

I think this is the best question: Who is this page for?

If you want to mention all these organisations we should also add Intens because we play poker with them and Synergy because we organise the BIT-event together, just like all the study associations of the beta faculty. And Olympus. And the SOFv. And the BBB. But does it make the website better?

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Jan 24, 2019, 20:54

Oh, please don't use the template for technical changes. This is a feature. It changes the behaviour of the website.

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Jan 27, 2019, 19:09

The page is for anyone looking for organisations that have a close relation with Thalia in terms of regular contact, or organisations that have to do anything with computing science.

Organisations like Intens and Synergy aren't included in this list as they only get in touch with Thalia once a year, and don't have anything to do with computing science other than that.

As we already have a list containing brother and sister associations (including Olympus and the BBB), we think it's nice to add other organisations to this list as well. They could offer people that do visit the existing page as much relevant info as that brother and sister associations currently do.

I think merging the page dedicated to student participation is a good idea. It centralizes info on the website. 

About the WISO: I agree that this contact is indeed only relevant for boards. I think it's a good idea to replace this one with the SOFv page that Sebastiaan mentioned.

Furthermore, naming the page 'Contacts' instead of related organisations might indeed be better when merging the page with the student participation page.

Thanks in advance!

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Jan 27, 2019, 19:25

> Organisations like Intens and Synergy aren't included in this list as they only get in touch with Thalia once a year, and don't have anything to do with computing science other than that.

So you want to keep it CS-only. Because SNiC and activities with RWoCS are also only once a year ;)

> About the WISO: I agree that this contact is indeed only relevant for boards. I think it's a good idea to replace this one with the SOFv page that Sebastiaan mentioned.

Isn't the SOFv also only relevant for boards? I'd say those are similar organisations, the meetings of the one are just a lot more fun than the meetings of the other.

> Furthermore, naming the page 'Contacts' instead of related organisations might indeed be better when merging the page with the student participation page.

I'm not in favour of merging the information on the student participation page with a page that consists only of hyperlinks to external websites. The contents differ quite a lot (explanations vs lists) and they're located (on purpose) in different parts of the navigation tree (education vs association).

If can agree that we don't do that and only add relevant organisations for potential members we can definitely put this in the next release :)

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Jan 27, 2019, 22:12

(at)rwechel do you still want to keep those contacts in the reference document? 

I do not think we want links to contacts without explanation on one page, whilst having explanation without links of those same organizations on another part of the website. It would be better to have one place with all info (for example by adding those links to the reference document).

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Jan 29, 2019, 11:04

> Isn't the SOFv also only relevant for boards? I'd say those are similar organisations, the meetings of the one are just a lot more fun than the meetings of the other.

Now that I think of it, the SOFv is less relevant for members than I thought at first. Let's leave it out.

> I'm not in favour of merging the information on the student participation page with a page that consists only of hyperlinks to external websites. The contents differ quite a lot (explanations vs lists) and they're located (on purpose) in different parts of the navigation tree (education vs association).

I will leave this up to you. I agree with your point, however centralized info might be clearer and more useful to website visitors.

> I do not think we want links to contacts without explanation on one page, whilst having explanation without links of those same organizations on another part of the website. It would be better to have one place with all info (for example by adding those links to the reference document).

Though I understand your point, I think this wouldn't be a reason to leave the contacts out of the webpage, or include links in the document. Both seem a bit weird to me. What if we add a text to the webpage saying "For more information about these contacts, see the reference document".

~~Sorry for the awful quoting btw, apparently I am a noob.~~ 

Edit: Quotes fixed by Joren.

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Jan 29, 2019, 11:31

> Now that I think of it, the SOFv is less relevant for members than I thought at first. Let's leave it out.

Please specify the current links you would like to see in the description.

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Jan 29, 2019, 11:43

As you said yourself "centralized info might be clearer and more useful to website visitors". That's why I like to see descriptions and links together. The student participation page also has descriptions and links. Users will find all info on one page and will not be sent from one page to another to find the desired information.

I also do not understand the selectiveness in organisations. On the one hand Synergy and Intens are not included and on the other hand RWoCS and SNiC are included, while Thalia has more contact with Synergy than with RWoCS.

I propose we move all contact descriptions from the reference document about all organizations to one contacts page that contains a description and links to all organisations (including SOFv, WiSO and student participation).  
This way:
- All contact information is easy to find because it is not scrambled throughout different pages/documents on the website.
- The information is easy to maintain, because there is only one page.
- There is no discussion on who should be added to the page, because all relevant parties are included.
- This replaces the reference document, which is hard to maintain (for example the Translacie needs to translate it) and easy to forget.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Jan 29, 2019, 11:44

> This replaces the reference document, which is hard to maintain (for example the Translacie needs to translate it) and easy to forget.

But probably better than having a page on the website that is hard to maintain and will only be updated in a release.

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Feb 2, 2019, 14:52

> I propose we move all contact descriptions from the reference document about all organizations to one contacts page that contains a description and links to all organisations (including SOFv, WiSO and student participation).  
This way:
> - All contact information is easy to find because it is not scrambled throughout different pages/documents on the website.
> - The information is easy to maintain, because there is only one page.
> - There is no discussion on who should be added to the page, because all relevant parties are included.
> - This replaces the reference document, which is hard to maintain (for example the Translacie needs to translate it) and easy to forget.

This sounds like an idea that works perfectly. The visibility of the information contained in the document is way better this way. Shall I create a new issue stating exactly what should happen?
The referral document also contains information about committees. I will revise every committee description on the website using the document, so this information will not get lost.

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Feb 9, 2019, 15:23

> This sounds like an idea that works perfectly. The visibility of the information contained in the document is way better this way. Shall I create a new issue stating exactly what should happen?
The referral document also contains information about committees. I will revise every committee description on the website using the document, so this information will not get lost.


Did you guys discuss this yet?

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Mar 22, 2019, 22:35

Completely missed this. I'm not sure we reached a conclusion. I still think we should keep the education page. I'll make sure to add it to the agenda for after the exams.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Mar 22, 2019, 22:35

changed due date to April 10, 2019

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Apr 10, 2019, 21:23

(at)rwechel, @se-bastiaan and I talked about this. Our conclusion is to make the page dynamic. That way the board could add contacts with a name, link and description and we do not need to change the info on the page or the reference document.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Apr 13, 2019, 12:42

removed due date

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Jul 10, 2019, 17:50

Updated (at)tvalten on this issue as I'll soon have nothing to do with this anymore

**Comment by thaliawww-service** [comment]
In GitLab by @lukovdm on Sep 18, 2019, 21:34

We think this is feature creep and would like to close this issue.

**Comment by thaliawww-service** [comment]
In GitLab by @lukovdm on Sep 18, 2019, 21:34

closed

**Comment by thaliawww-service** [comment]
In GitLab by rwechel on Sep 18, 2019, 21:39

Seriously?

**Comment by thaliawww-service** [comment]
In GitLab by @joren485 on Sep 18, 2019, 21:40

I also think it is a bit strange that this is simply closed as "feature creep" without explanation after a very long discussion.

**Comment by thaliawww-service** [comment]
In GitLab by @se-bastiaan on Sep 18, 2019, 21:54

We discussed this as being a good option for solving a problem that in our eyes does not really exist. The current option works fine and the only thing this issue does is add more code to maintain. The number of times these documents/pages change is low, so the maintainability of the current solution is better than adding a level of abstraction that also needs maintaining.
